vendor.mod

```diff
@@ -186,7 +194,7 @@ require (
 	github.com/package-url/packageurl-go v0.1.1-0.20220428063043-89078438f170 // indirect
 	github.com/philhofer/fwd v1.1.2 // indirect
 	github.com/planetscale/vtprotobuf v0.6.1-0.20240319094008-0393e58bdf10 // indirect
-	github.com/pmezard/go-difflib v1.0.0 // indirect
+	github.com/pmezard/go-difflib v1.0.1-0.20181226105442-5d4384ee4fb2 // indirect
```

Comment:
Same for this one 😡 who decided to YOLO master for _no reason at all_ ? https://github.com/pmezard/go-difflib/compare/v1.0.0...5d4384ee4fb2527b0a1256a821ebfc92f91efefc

Reply:
Looks like it may be nydus;

```
go mod graph | grep ' github.com/pmezard/go-difflib@v1.0.1'
github.com/docker/docker github.com/pmezard/go-difflib@v1.0.1-0.20181226105442-5d4384ee4fb2
github.com/containerd/containerd/v2@v2.0.0 github.com/pmezard/go-difflib@v1.0.1-0.20181226105442-5d4384ee4fb2
github.com/containerd/nydus-snapshotter@v0.14.1-0.20241016012921-e55ae70fd45a github.com/pmezard/go-difflib@v1.0.1-0.20181226105442-5d4384ee4fb2
github.com/moby/buildkit@v0.18.2 github.com/pmezard/go-difflib@v1.0.1-0.20181226105442-5d4384ee4fb2
```

Reply:
No! It's containerd, through cri-api; came in through this PR:

- https://github.com/containerd/containerd/pull/9737

```bash
go mod graph | grep ' github.com/davecgh/go-spew@v1.1.2'
github.com/containerd/containerd/v2 github.com/davecgh/go-spew@v1.1.2-0.20180830191138-d8f796af33cc
k8s.io/cri-api@v0.31.0-alpha.0.0.20240529224029-3a66d9d86654 github.com/davecgh/go-spew@v1.1.2-0.20180830191138-d8f796af33cc

go mod graph | grep ' github.com/pmezard/go-difflib@v1.0.1'
github.com/containerd/containerd/v2 github.com/pmezard/go-difflib@v1.0.1-0.20181226105442-5d4384ee4fb2
k8s.io/cri-api@v0.31.0-alpha.0.0.20240529224029-3a66d9d86654 github.com/pmezard/go-difflib@v1.0.1-0.20181226105442-5d4384ee4fb2
```

Reply:
And in the cri-api repo it is hard to see the pull request it was brought it in with to see if it was intentional or discussed. We can bump it to a tag in containerd if we want.

Reply:
- And that came in through https://github.com/kubernetes/cri-api/commit/1171b1d7d1a2b443aded68a00326e24be4fbb874 -> https://github.com/kubernetes/kubernetes/pull/123339
- Which came in through https://github.com/kubernetes/kubernetes/pull/123339#discussion_r1618125911
- Which was because of https://github.com/kubernetes-sigs/kustomize/pull/5615
- And https://github.com/spf13/viper/pull/1574

Looks like someone else also went down the rabbit-hole, and found the culprit, and reverted, but that's gonna take some time to get out of :disappointed:

- https://github.com/spf13/viper/pull/1861

Reply:
Opened PRs in kubernetes and containerd to get back to released versions;

- https://github.com/kubernetes-sigs/kustomize/pull/5830
- https://github.com/containerd/containerd/pull/11220